spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -53,6 +53,19 @@
           expect(json_data.size).to eq(2)
         end
       end
+
+      describe 'filters by status' do
+        it 'filters by single status' do
+          get :index, params: { filter: { status_eq: another_transcription.status_before_type_cast } }
+          expect(json_data.size).to eq(1)
+          expect(json_data.first["attributes"]["status"]).to eq(another_transcription.status)
+        end
+
+        it 'filters by multiple statuses' do
+          get :index, params: { filter: { status_in: "#{another_transcription.status_before_type_cast},#{separate_transcription.status_before_type_cast}" } }
```

Comment:
Line gets a bit long there, maybe break it up for readability.

```suggestion
          status_filter = "#{another_transcription.status_before_type_cast},#{separate_transcription.status_before_type_cast}"
          get :index, params: { filter: { status_in: status_filter } }
```

Reply:
makes sense 100 percento. good call.